.prettierrc.js

```diff
@@ -1,9 +1,3 @@
 module.exports = {
```

Comment:
All of these except semi are default values

(And print width being 80 instead of 88)